### ISSUE TITLE
Migration from RAFT to BFT: IT: check mixed orderer

### ISSUE THREAD

**Issue by tock-ibm** [hyperledger/fabric#4643]
Check what happens to a Raft-based orderering service that has two channels, A & B.
Channel A is migrated to BFT but B does not.
If the ordering service exits maintenance mode, do both channels continue to function properly? 
We do not plan to support this scenario in production, but we would like to experiment and see what happens.

**Comment by dviejokfs** [comment]
Tested it at the end of this [online meetup about deploying Hyperledger Fabric 3.0 using the bevel-operator-fabric](https://www.youtube.com/watch?v=4taLwa_pl9U&t=2850s)

**Comment by tock-ibm** [comment]
> Tested it at the end of this [online meetup about deploying Hyperledger Fabric 3.0 using the bevel-operator-fabric](https://www.youtube.com/watch?v=4taLwa_pl9U&t=2850s)

@dviejokfs Thanks! Could you summarize here what exactly you tested and how, and what were the results? 

Our decision right now is to not officially support mixed orderers, i.e. an orderer with channels in both etcdraft & BFT. This does not mean it will not work, we just do not want to commit to it working and devote testing and development resource to it. However it would be interesting and important to know what actually happens in this case.
 
PS did you have the chance to check a mixed peer? #4642
